quesma/queryparser/aggregation_parser.go

```diff
@@ -228,8 +227,9 @@ func (b *aggrQueryBuilder) buildMetricsAggregation(metricsAggr metricsAggregatio
 			)
 			firstCountExp := model.NewFunction("count", ifExp)
```

Comment:
While there, wouldn't be easier just to use `countIf` function?
https://clickhouse.com/docs/en/sql-reference/aggregate-functions/combinators

It sounds simpler and likely ClickHouse could optimize it better.

Reply:
We may also do TODO, to replace and centralize second count(*).